$NetBSD: patch-ab,v 1.3 2011/12/12 16:26:37 dholland Exp $

- remove bashisms
- correct strange belief that 32-bit asm works on x86_64

--- configure.orig	2007-10-17 20:00:50.000000000 +0000
+++ configure
@@ -6024,7 +6024,7 @@ fi
 
 
 I386="no"
-for i in i386 i486 i586 i686 i786 x86_64 ; do
+for i in i386 i486 i586 i686 i786 ; do
   if test "$i" = "$target_cpu" ; then
     I386="yes"
   fi;
@@ -6392,7 +6392,7 @@ echo $ECHO_N "checking for target specif
 # Check whether --enable-target-opt was given.
 if test "${enable_target_opt+set}" = set; then
   enableval=$enable_target_opt;
-  if test "$I386" == "yes" && test "$enableval" != "no" ; then
+  if test "$I386" = "yes" && test "$enableval" != "no" ; then
     TARGET_OPT=yes
     { echo "$as_me:$LINENO: result: enabled" >&5
 echo "${ECHO_T}enabled" >&6; }
@@ -6404,7 +6404,7 @@ echo "${ECHO_T}disabled" >&6; }
 
 else
 
-  if test "$I386" == "yes" ; then
+  if test "$I386" = "yes" ; then
     TARGET_OPT=yes
     { echo "$as_me:$LINENO: result: enabled" >&5
 echo "${ECHO_T}enabled" >&6; }
